src/main/java/kitchenpos/menus/tobe/domain/MenuProductQuantity.java

```diff
@@ -0,0 +1,52 @@
+package kitchenpos.menus.tobe.domain;
+
+import kitchenpos.menus.exception.MenuErrorCode;
+import kitchenpos.menus.exception.MenuProductQuantityException;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+
+@Embeddable
+public class MenuProductQuantity {
+    @Column(name = "quantity", nullable = false)
+    private long value;
+
+    protected MenuProductQuantity() {
+
+    }
+
+    public MenuProductQuantity(long value) {
+        if (isLessThanZero(value)) {
+            throw new MenuProductQuantityException(MenuErrorCode.QUANTITY_IS_GREATER_THAN_ZERO);
+        }
+        this.value = value;
+    }
+
+    private boolean isLessThanZero(long input) {
+        return input < 0;
+    }
+
+
+    public long getValue() {
+        return value;
+    }
+
+
```

Comment:
공백도 의미있게 사용하면 좋습니다. 전체적으로 컨벤션은 확인해 보시면 좋을 것 같네요. 

```suggestion
    private boolean isLessThanZero(long input) {
        return input < 0;
    }

    public long getValue() {
        return value;
    }

```